Sources/SwiftKotlinFramework/KotlinTokenizer.swift

```diff
@@ -619,28 +885,58 @@ public class KotlinTokenizer: SwiftTokenizer {
         case let .interpolatedString(_, rawText):
             return tokenizeInterpolatedString(rawText, node: expression)
         case .array(let exprs):
-            return
-                expression.newToken(.identifier, "listOf") +
-                expression.newToken(.startOfScope, "(") +
-                exprs.map { tokenize($0) }.joined(token: expression.newToken(.delimiter, ", ")) +
-                expression.newToken(.endOfScope, ")")
+            // Trying to output example mutableListOf<String> if it's a declartion, or mutableListOf("xx", "yy") if it's an instance with values, not great
+            var hasType = false
+            for e in exprs {
+                switch e {
+                case let _ as IdentifierExpression:
+                    hasType = true
+                default:
+                    break
+                }
+            }
+            if hasType && exprs.count == 1 && !firstCharIsUpper(str:exprs[0].description) {
+                hasType = false
+            }
+            let middle = exprs.map { tokenize($0) }.joined(token: expression.newToken(.delimiter, ", "))
+            let mutable = [ expression.newToken(.identifier, "mutableListOf") ]
+            if hasType {
+                return
+                    mutable +
+                    [ expression.newToken(.startOfScope, "<") ] +
+                    middle +
+                    [ expression.newToken(.endOfScope, ">") ]
+            } else {
+                return
+                    mutable +
+                        [ expression.newToken(.startOfScope, "(") ] +
+                        middle +
+                        [ expression.newToken(.endOfScope, ")") ]
+            }
+            // have to output mutableMap since all swift arrays are mutable. Not great.
```

Comment:
Actually, swift arrays are non-mutable when `let` and mutable when `var` because they are value types. The attempt to map it to `mutableList` instead of `list` is nice but it should be dependent on the variable type.